railties/lib/rails/engine.rb

```diff
@@ -619,7 +619,7 @@ def load_seed
       next if is_a?(Rails::Application)
 
       fixtures = config.root.join("test", "fixtures")
-      if fixtures.exist? && fixtures.to_s.start_with?(Rails.root.to_s)
+      if fixtures.exist? && fixtures.to_s.start_with?(Rails.root.to_s) && !fixtures.to_s.start_with?(Rails.root.join("vendor").to_s)
```

Comment:
Sorry I don't have a better alternative off-hand, but I wonder if 3 conditionals on a single line is a good reason to pull this into a separate method?

Reply:
@zzak Changed, the testing failure is [not relative](https://buildkite.com/rails/rails/builds/96731#018865e7-ef11-4463-b0ab-cda513be9d7f/1102-1123) with this PR. (https://saucelabs.com/rest/v1/rubyonrails/tunnels - 500 (Internal Server Error) Error)

Reply:
@Eric-Guo Nice! Maybe "fixtures_in..." plural?

Reply:
Changed.